src/documentation/fares_internal_format.md

```diff
@@ -0,0 +1,35 @@
+# Representing fares in Navitia Transit Model
+## Overview
+This document describes how fare structures are modeled in Navitia Transit Model. The internal model is composed of the followning objects:
+- Ticket
+- Rules
+
+## Model details
+### Ticket
+This object contains the necessary information to specify a ticket.
+
+Property | Type | Required | Description
+--- | --- | --- | ---
+id | UUID | Yes | Unique identifier
+input_data_format | Enum | Yes | Source of the fare data, fixed value `netex_fares_nl`.
+input_ticket_id | String | No | Id of the ticket found in the source data.
+start_date | Date | Yes | Start date for the validity period of the ticket price.
+end_date | Date | Yes | End date for the validity period of the ticket price. This date is included in the validity period interval.
+currency_type | String | Yes | The currency used to pay the ticket. The [ISO 4217](https://en.wikipedia.org/wiki/ISO_4217#Active_codes) currency codes are used.
+price | Float | Yes | The total price of the ticket in the currency specified by `currency_type`, including any additional fee, if any (e.g. boarding fee).
+validity_duration | Integer | No | Validity duration of the ticket in seconds.
+transfers | Integer | No | Number of transfers allowed for the ticket.
+
+### Rules
+This object specifies how a `Ticket` is used and applied in a Navitia trip.
```

Comment:
Should we specify the meaning of full-empty destination fields?

Reply:
Instead of  `Navitia trip`, I think `Navitia journey` is mode accurate.
Maybe specifying an `OD Rule` for OD fares will be easier to understand and use.